### ISSUE TITLE
Null Reference Exception while editing Debug Profiles

### ISSUE THREAD

**Issue by Forge36** [dotnet/project-system#8435]
## Visual Studio Version
Microsoft Visual Studio Professional 2022 (64-bit) - Current
Version 17.2.6

## Summary
Null Reference exception popup when pressing F2 without a selection in Debug Launch Profiles UI

## Steps to Reproduce

1. Click Run dropdown menu > [Project] Debug Properies
2. Select a configuration 
3. Rename with F2
4. Notice after rename nothing is selected
5. Press F2 again (see error dialog)

## Expected Behavior
At step 4 I'd expect the previous profile to be selected.
Baring that: I'd expect step 5 do "do nothing" lacking a current selection.

## Actual Behavior
Error dialog is shown:
```
---------------------------

---------------------------
System.NullReferenceException: Object reference not set to an instance of an object.

   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.LaunchProfilesWindowViewModel.<>c__DisplayClass4_1.<.ctor>b__4(LaunchProfileViewModel profile)

   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.DelegateCommand`1.Execute(Object parameter)
---------------------------
OK   
---------------------------
```

## User Impact
Momentary confusion.

**Comment by ocallesp** [comment]
Repro
![reproGithubBug8435](https://user-images.githubusercontent.com/510598/187838715-a6a30f5a-c046-4a68-9339-39f9032a08d6.gif)

**Comment by adamint** [comment]
There is an accessibility issue in addition to the logical bug here. Focus needs to return to the profile control after pressing enter.

**Comment by adamint** [comment]
PR tracked in https://devdiv.visualstudio.com/DevDiv/_git/CPS/pullrequest/423377

**Comment by adamint** [comment]
@Forge36 ty for reporting, this will be fixed in the next VS preview release
